pipelines/skylab/build_indices/BuildIndices.wdl

```diff
@@ -106,365 +26,70 @@ task BuildStarSingleNucleus {
     set -eo pipefail
 
     python3 /script/modify_gtf.py  \
-    --input-gtf ~{references.annotation_gtf} \
+    --input-gtf ~{annotation_gtf} \
     --output-gtf ~{annotation_gtf_modified} \
     --biotypes ~{biotypes}
 
+    python3  /script/add-introns-to-gtf.py   --input-gtf ~{annotation_gtf_modified}  --output-gtf ~{annotation_gtf_introns}
+
     mkdir star
     STAR --runMode genomeGenerate \
     --genomeDir star \
-    --genomeFastaFiles ~{references.genome_fa} \
+    --genomeFastaFiles ~{genome_fa} \
     --sjdbGTFfile ~{annotation_gtf_modified} \
     --sjdbOverhang 100 \
     --runThreadN 16
 
     tar -cvf ~{star_index_name} star
 
-    python3  /script/add-introns-to-gtf.py   --input-gtf ~{annotation_gtf_modified}  --output-gtf ~{annotation_gtf_introns}
   >>>
 
   output {
     File star_index = star_index_name
     File annotation_gtf_modified_introns = annotation_gtf_introns
-    References modified_references = object {
-             genome_fa: references.genome_fa,
-             annotation_gtf: annotation_gtf_modified
-           }
+    File modified_annotation_gtf = annotation_gtf_modified
   }
-  
+
   runtime {
-    docker: "us.gcr.io/broad-gotc-prod/build-indices:1.0.0-2.7.10a-1663605340"
+    docker: "us.gcr.io/broad-gotc-prod/build-indices:1.0.0-2.7.10a-1671132408"
```

Comment:
If this is the finalized version of the docker we are going to merge into develop, do you mind adding a line to the .tsv? In dockers/skylab/build-indices/docker_versions.tsv

Reply:
No, I just updated the docker with changes Jess recommended and added the a new tag to the wdl. Thank you for reminding me to update the tsv!